clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -189,6 +192,11 @@ abstract class AbstractEurekaSupport {
         AbstractEurekaSupport.log.info("[$phaseName] - Failed marking discovery $discoveryStatus.value for instances ${errors}")
       }
     }
+    if (skipped) {
+      task.addResultObjects([
+        ["discoverySkippedInstanceIds": instanceIds]
```

Comment:
Orca's `WaitForRequiredInstancesDownTask` operates over an `instanceIdsToDisable` list provided via kato results. We've observed task timeouts caused by instances that were skipped by `updateDiscoveryStatusForInstances` due to a 404 but that never pass the `someAreDownAndNoneAreUp` test. When this is the case as part of a RRB deploy stage where < 100% of the prior asg is being disabled, we can potentially punt such instances to a latter rolling group, or explicitly retry from orca.

Reply:
👍